deploy/osd-scanning/20-osd-scanning-logger-Daemonset.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: logger
+  namespace: openshift-scanning
+spec:
+  selector:
+    matchLabels:
+      name: logger
+  template:
+    metadata:
+      labels:
+        name: logger
+    spec:
+      containers:
+      - env:
+        - name: OO_PAUSE_ON_START
+          value: "false"
+        - name: LOG_WRITER_URL
+          value: http://logger.openshift-scanning.svc:8080/api/log
+        - name: SCAN_LOG_FILE
+          value: /host/var/log/openshift_managed_malware_scan.log
+        - name: POD_LOG_FILE
+          value: /host/var/log/openshift_managed_pod_creation.log
+        image: quay.io/dedgar/pod-logger:v0.0.10
```

Comment:
Can we move this image to a more official repo or possibly just use the upstream image?